lib/ansible/modules/remote_management/redfish/redfish_command.py

```diff
@@ -147,6 +147,7 @@
 CATEGORY_COMMANDS_ALL = {
     "Systems": ["PowerOn", "PowerForceOff", "PowerGracefulRestart",
                 "PowerGracefulShutdown", "PowerReboot", "SetOneTimeBoot"],
+    "Chassis": ["IndicatorLedOn", "IndicatorLedOff", "IndicatorLedBlink"],
```

Comment:
Would be good to add an example of at least one of these in the EXAMPLES doc string above.